lib/src/screens/supa_verify_phone.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:supabase_auth_ui/src/utils/supabase_auth_ui.dart';
+import 'package:supabase_auth_ui/src/utils/constants.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+class SupaVerifyPhone extends StatefulWidget {
+  final String? redirectUrl;
+
+  const SupaVerifyPhone({Key? key, this.redirectUrl}) : super(key: key);
+
+  @override
+  State<SupaVerifyPhone> createState() => _SupaVerifyPhoneState();
+}
+
+class _SupaVerifyPhoneState extends State<SupaVerifyPhone> {
+  Map? data;
+  final _formKey = GlobalKey<FormState>();
+  final _code = TextEditingController();
+
+  SupabaseAuthUi supaAuth = SupabaseAuthUi();
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _code.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var args = ModalRoute.of(context)?.settings.arguments;
+    if (args != null) data = args as Map;
+    print(data);
+    return Form(
+      autovalidateMode: AutovalidateMode.onUserInteraction,
+      key: _formKey,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          TextFormField(
+            validator: (value) {
+              if (value == null ||
+                  value.isEmpty ) {
+                return 'Please enter a valid email address';
```

Comment:
```suggestion
                return 'Please enter your one time password.';
```

Reply:
I think its clearer to say enter the code sent than a password, just to minimise possible confusion for users.

Reply:
Sure, I am not attached to this particular line at all, so please feel free to change it to whatever you think is best!